doc/sphinx/source/api/esmvaltool.diag_scripts.rst

```diff
@@ -1,8 +0,0 @@
-Diagnostic scripts
```

Comment:
FYI: I removed this page since it contains now useful information. In the current doc it looks like this:

![grafik](https://user-images.githubusercontent.com/32543114/168551368-4b15a46a-6a2c-4df4-86f0-db9a8b85c9b7.png)